codegen/service/testdata/convert_functions.go

```diff
@@ -197,19 +191,15 @@ func marshalObjectFieldToObjectFieldTExt(v *ObjectField) *testdata.ObjectFieldT
 		Bytes:   v.Bytes,
 		String:  v.String,
 	}
-	if v.Array != nil {
-		res.Array = make([]bool, len(v.Array))
-		for i, val := range v.Array {
-			res.Array[i] = val
-		}
+	res.Array = make([]bool, len(v.Array))
+	for i, val := range v.Array {
+		res.Array[i] = val
 	}
-	if v.Map != nil {
-		res.Map = make(map[string]bool, len(v.Map))
-		for key, val := range v.Map {
-			tk := key
-			tv := val
-			res.Map[tk] = tv
-		}
+	res.Map = make(map[string]bool, len(v.Map))
```

Comment:
This code (same above) will create a non-nil attribute instead of keeping it nil if the original is nil.

Reply:
These are required attributes. So they must be non-nil correct?